asset/css/search-bar.less

```diff
@@ -193,27 +205,32 @@
   display: flex;
   position: relative; // Required for the suggestions
 
+  .filter-input:focus {
+    outline-offset: -1px;
+  }
+
   .filter-input-area {
     overflow: auto hidden;
     overflow-x: overlay; // Not invalid, but proprietary feature by chrome/webkit
     display: flex;
     width: 100%;
-    padding: .3em .3em .6em .4em;
+    padding: 2/12em;
 
     label input {
-      line-height: 1.5;
+      line-height: 20/12;
     }
 
     // Lets inputs grow based on their contents, Inspired by https://css-tricks.com/auto-growing-inputs-textareas/
     label {
       position: relative;
       display: inline-block;
       min-width: 2em;
+      line-height: 20/12;
```

Comment:
Instead of re-adding the same comment from above, consider to use a variable here. Then add the comment to the variable definition.

Reply:
I removed the `line-height` property for the label, as it was not necessary.

Reply:
✅